server/testutils/factories/premises.ts

```diff
@@ -22,7 +36,7 @@ export default Factory.define<Premises>(() => ({
   characteristics: unique(
     referenceDataFactory.characteristic('premises').buildList(faker.datatype.number({ min: 1, max: 5 })),
   ),
-  status: faker.helpers.arrayElement(['active', 'archived']),
+  status: faker.helpers.arrayElement(['active', 'archived'] as const),
```

Comment:
Ah, this won't influence the underlying `Premises` type - this helps the typechecker figure out that this is an array *of these particular strings* rather than just and array of strings, and so can a value from the array can be assigned to the type `'pending' | 'active' | 'archived'`.

It worked before without the `as const` but now needs it, I think because now we're defining our own `PremisesFactory` the typecheck needs a bit more help working though the layers of indirection